src/presentation/http/http-server.ts

```diff
@@ -50,7 +43,7 @@ export default class HttpServer implements API {
    * @param domainServices - instances of domain services
    */
   public async run(domainServices: DomainServices): Promise<void> {
-    const middlewares = initMiddlewares();
+    const middlewares = initMiddlewares(domainServices.authService);
```

Comment:
seems not scalable, why only authService available for all middlewares?

Reply:
You suggest to pass all services?

Reply:
done